webpages/settings/index.js

```diff
@@ -540,7 +586,15 @@ const vue = (window.vue = new Vue({
         this.isOpen = false;
       }
     },
+    closePickers(e) {
+      for (let child of this.$children) {
```

Comment:
Is `child` guaranteed to be a color picker component?

Reply:
No. We just have no other components on the page.

Reply:
Also, no other components will have all of the correct methods it checks.